platinum-bluetooth-device.html

```diff
@@ -83,7 +83,33 @@
          */
         servicesFilter: {
           type: Array,
-          observer: '_servicesFilterChanged'
+          observer: '_filterChanged'
+        },
+
+        /**
+         * Device name filter.
+         */
+        nameFilter: {
+          type: String,
+          observer: '_filterChanged'
+        },
+
+        /**
+         * Device name prefix filter.
+         */
+        namePrefixFilter: {
+          type: String,
+          observer: '_filterChanged'
+        },
+
+        /**
+         * Optional Bluetooth GATT services to get access to. You may provide
+         * either the full Bluetooth UUID as a string or a short 16- or 32-bit
+         * form as integers like 0x180d.
+         */
+        optionalServices: {
```

Comment:
Could you remove optionalServices as it's not implemented yet?
I just want to lower down developers expectation there ;)